.circleci/config.yml

```diff
@@ -1,129 +1,137 @@
+---
 version: 2.1
-
 release_tag_only: &release_tag_only
   filters:
     tags:
-      only: /^v[0-9]+\.[0-9]+\.[0-9]+/
+      only: "/^v[0-9]+\\.[0-9]+\\.[0-9]+/"
     branches:
-      ignore: /.*/
+      ignore: "/.*/"
 
 executors:
   docker-amd64:
     parameters:
       image:
         type: string
     docker:
-      - image: << parameters.image >>
+    - image: "<< parameters.image >>"
     resource_class: xlarge
   docker-arm64:
     parameters:
       image:
         type: string
     docker:
-      - image: << parameters.image >>
+    - image: "<< parameters.image >>"
     resource_class: arm.xlarge
 
 jobs:
-  build:
+  build_amd64:
     parameters:
-      # `build-image` is the docker image, with tag, in which the module will
-      # be built. It's of the form
-      # "datadog/docker-library:nginx-datadog-build-$base_image_without_colons".
-      build-image:
-        type: string
-      # `base-image` is the docker image on which the build-image is based.
-      # This parameter is extracted from nginx-version-info by the release
-      # script, so that it knows how to name the built shared object.
-      base-image:
-        type: string
-        default: ""
-      # `nginx-version` is the release version of the nginx source code to
-      # include in the built module. It's of the form
-      # "$major.$minor.$patch".
       nginx-version:
         type: string
-      arch:
-        type: string
-        default: "amd64"
-    executor:
-      name: docker-<< parameters.arch >>
-      image: << parameters.build-image >>
+      waf:
+        type: enum
+        enum:
+        - 'ON'
+        - 'OFF'
+    steps:
+    - checkout
+    - run: git submodule sync && git submodule update --init --recursive
+    - run:
+        command: 'make build-musl'
```

Comment:
It builds against musl in a linux VM. I think "build" should be reserved for using the toolchain on the host machine. It would be strange if "make build" would generate a library for Linux when run on Mac OS.

Reply:
Wow I was not focused when I wrote that. I meant, now, musl is the default libc?